lib/editor/components/EditorInput.js

```diff
@@ -300,6 +300,8 @@ export default class EditorInput extends Component<Props> {
           </span>
         )
       case 'DROPDOWN':
+        const standard = 'Standard'
+        const extended = 'Extended'
         return (
```

Comment:
To fix the flow type issues and simplify things, I would add something like the following:
```javascript
if (!field.options) {
  console.warn(`Field `field.name` does not have options configured, but is marked as inputType=DROPDOWN`)
  return null
}
```

Reply:
Where do you see the flow issues?  When I load the project (before typing any code, I get these kind of squigglies from the beginning of the render method:
![image](https://user-images.githubusercontent.com/4264701/114189718-68cba580-9942-11eb-8d3f-7f99253166f5.png)

Reply:
When I run `flow status` I get an error:
![image](https://user-images.githubusercontent.com/4264701/114190460-3d958600-9943-11eb-8393-94603ebb983f.png)

Reply:
The flow errors are showing up in the GitHub builds: https://github.com/ibi-group/datatools-ui/pull/656/checks?check_run_id=2298915284

Locally, you should run `yarn flow`.